### DISCUSSION TITLE
Scalar should read OpenApi Info section

### DISCUSSION THREAD

**Discussion by alihankuzucuk** [scalar/scalar#4004] [Q&A]
While creating the OpenApi documentation file with .NET Core, I fill OpenApiInfo object which is for "info" section.

I can see the "info" information of my service in my OpenApi json file, but it does not reflect this in the Scalar UI interface.

![screenshot](https://github.com/user-attachments/assets/c5910c03-9864-4dff-af48-63273aee5ae7)

**Attention:**
OpenApiInfo object in .NET Core is used not only for "**contact**" information but also for "**license**" and "**terms of service**" information.

**Comment by hanspagel** [comment]
You’re right, a few attributes aren’t rendered yet. We’ll add them. ✌️

**Reply by vtachkov** [reply]
Just wondering if this is still on the roadmap. I'm currently using redoc which does render the contact info at the top of the rendered document, but I need to migrate to another tool. Scalar is my top choice, but when I was giving it a test run, I noticed that it didn't support this relatively simple feature.
